PERSONAL_DATA_DISCLOSURE.md

```diff
@@ -0,0 +1,84 @@
+_Last Updated: (date form was last changed)_
+_Last Reviewed: (date form was last reviewed for accuracy)_
```

Comment:
On the BE side, we are providing the actual dates too. 
https://github.com/EBSCOIS/mod-lists/blob/master/PERSONAL_DATA_DISCLOSURE.md?plain=1
For instance, 
```
_Last Updated: 2023-08-17_
_Last Reviewed: 2023-08-17_
```

Reply:
Actually, those aren't fields for GDPR, but I do need to fill them out. One for when I last updated this document and the other for reviewing.